clog/logger.go

```diff
@@ -253,3 +260,24 @@ func Singleton() *builder {
 func Flush(ctx context.Context) {
 	_ = Ctx(ctx).zsl.Sync()
 }
+
+// Inherit propagates the clog client from one context to another.  This is particularly
+// useful for taking an initialized context from a main() func and ensuring the logger
+// is available for request-bound conetxts, such as in a http server pattern.
+//
+// If the 'to' context already contains an initialized logger, no change is made.
```

Comment:
> If the 'to' context already contains an initialized logger, no change is made.

Is there a reason why we want to preserve the `to` logger? Asking because `Inherit` by it's name implies unconditional inheritance of `from` logger. Developers may call `Inherit` and expect `from` logger to always overwrite any possible `to` logger. Same goes for `otel` client  and `ctats` inheritance.

Reply:
hmm, good question.  How about I make it an optional toggle?

Reply:
Good idea!